src/screens/ChatDetailPage/index.js

```diff
@@ -208,7 +208,7 @@ const CustomDateHeader = () => null;
 export default withInteractionsManaged(ChatDetailPage);
 const styles = StyleSheet.create({
   date: {
-    color: COLORS.white,
+    color: COLORS.almostBlack,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color of the date text has been changed from white to almost black. This could potentially reduce readability if the background is dark. Please ensure that this color change doesn't affect the readability of the date text.
```diff
-    color: COLORS.white,
+    color: COLORS.almostBlack,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->